apps/test/unit/lib/levelbuilder/code-docs-editor/ProgrammingExpressionEditorTest.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+import {expect} from '../../../../util/reconfiguredChai';
+import ProgrammingExpressionEditor from '@cdo/apps/lib/levelbuilder/code-docs-editor/ProgrammingExpressionEditor';
+import {getStore} from '@cdo/apps/redux';
+import {Provider} from 'react-redux';
+import sinon from 'sinon';
+
+describe('ProgrammingExpressionEditor', () => {
+  let defaultProps, fetchSpy;
+
+  beforeEach(() => {
+    defaultProps = {
+      initialProgrammingExpression: {
+        id: 1,
+        name: 'Block',
+        key: 'block',
+        shortDescription: 'This is a short description.'
+      }
+    };
+    fetchSpy = sinon.stub(window, 'fetch');
+  });
+
+  afterEach(() => {
+    fetchSpy.restore();
+  });
+
+  it('renders default props', () => {
```

Comment:
could we have this try harder to describe the business logic, e.g. "input fields display initial values"?